src/mgr/DaemonServer.cc

```diff
@@ -2556,11 +2556,31 @@ bool DaemonServer::_handle_command(
     return true;
   }
 
+  // This section will be remove dlater, just for testing
+  auto finisher_val = py_modules.get_module(mod_name)->get_threshold_val();
+  auto queue_len = py_modules.get_mod_finisher_queue_len(mod_name);
+  ss << "Module '" << mod_name << "' has passed its finisher thread " << finisher_val <<" value"
+          "  QUEUE len" << queue_len;
+  dout(4) << ss.str() << dendl;
```

Comment:
this debug statement would seem to indicate we have passed the threshold which is not necessarily true

Reply:
As mentioned above in the comment, this is just added for testing.I will remove it later.

Reply:
Don't remove the debug statement (which can be useful), just word it appropriately and set a reasonable debug level.

Reply:
OK, I have reworded the debug statement.